api/graphql/schema.graphql

```diff
@@ -336,15 +336,26 @@ enum PoolCandidateSearchStatus {
     DONE
 }
 
+type EquityObject {
```

Comment:
I'm not sure `object` is the best suffix.  Naming is hard...  @Jerryescandon what are these?  EquityFilters? But filters sound like each will remove results and we're changing them to add results.  EquityAdditions?  EquityPreferences?  Maybe just plain-ole EmploymentEquity.